pkg/apis/config/v1alpha2/mesh_config.go

```diff
@@ -234,4 +234,7 @@ type FeatureFlags struct {
 
 	// EnableRetryPolicy defines if retry policy is enabled.
 	EnableRetryPolicy bool `json:"enableRetryPolicy"`
+
+	// EnableMeshRootCertificate defines if MRCs are used for certificate management.
```

Comment:
Can we add a sentence saying that this feature flag requires a control plane restart?

Reply:
Good idea. We'll also call this out and explain it in more depth in the docs.

Reply:
Updated!